apps/web-giddh/src/app/ledger/components/new-ledger-entry-panel/new-ledger-entry-panel.component.ts

```
@@ -487,7 +486,12 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
     private getAllDiscounts(callback?: () => void): void {
         this.settingsDiscountService.GetDiscounts().pipe(take(1)).subscribe(response => {
             if (response?.status === "success" && response?.body?.length > 0) {
-                this.discountsList.set(response?.body);
+                let discounts = response?.body;
+                discounts.map(discount => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add a return with a value to this callback.

(lint/suspicious/useIterableCallbackReturn)